rpc/backend/blocks.go

```diff
@@ -89,24 +86,71 @@ func (b *Backend) GetBlockByHash(hash common.Hash, fullTx bool) (map[string]inte
 		return nil, nil
 	}
 
-	blockRes, err := b.GetTendermintBlockResultByNumber(&resBlock.Block.Height)
+	blockRes, err := b.TendermintBlockResultByNumber(&resBlock.Block.Height)
 	if err != nil {
 		b.logger.Debug("failed to fetch block result from Tendermint", "block-hash", hash.String(), "error", err.Error())
 		return nil, nil
 	}
 
-	res, err := b.EthBlockFromTendermint(resBlock, blockRes, fullTx)
+	res, err := b.BlockFromTendermintBlock(resBlock, blockRes, fullTx)
 	if err != nil {
-		b.logger.Debug("EthBlockFromTendermint failed", "hash", hash, "error", err.Error())
+		b.logger.Debug("GetEthBlockFromTendermint failed", "hash", hash, "error", err.Error())
 		return nil, err
 	}
 
 	return res, nil
 }
 
-// GetTendermintBlockByNumber returns a Tendermint formatted block for a given
+// GetBlockTransactionCountByHash returns the number of transactions in the
+// block identified by hash.
+func (b *Backend) GetBlockTransactionCountByHash(hash common.Hash) *hexutil.Uint {
```

Comment:
I think it could be important to describe this are ETH transactions only in the name or comment.

Reply:
@facs95 Good point, I'll add it in the comment, as this function has the same name as the JSON-rpc endpoint